--- sbin/init.d/boot	Fri Mar 17 14:45:58 2000
+++ sbin/init.d/boot	Tue Apr  4 18:42:39 2000
@@ -12,6 +12,13 @@
 # first script to be executed from init on system startup
 #
 
+if [ -x /sbin/live-setup -a ! -d /S.u.S.E./bin ] ; then
+  ECHO_RETURN="\033[71G\033[32mdone\033[m"
+  echo -n "Integrating Live CD"
+  /sbin/live-setup || ECHO_RETURN="\033[71G\033[31m\033[1mfailed\033[m"
+  echo -e "$ECHO_RETURN"
+fi
+
 . /etc/rc.config
 echo "Running $0"
 
