--- sbin/init.d/boot	Wed Apr 26 15:38:18 2000
+++ sbin/init.d/boot	Wed Apr 26 15:39:18 2000
@@ -220,7 +220,7 @@
 #
 # clean up
 #
-rm -f /etc/mtab* /etc/nologin /nologin /fastboot
+rm -f /etc/nologin /nologin /fastboot
 
 #
 # initialize database for kerneld.  This should be done earlier, but
@@ -251,6 +251,7 @@
 ECHO_RETURN=$rc_done_up
 echo "Mounting local file systems..."
 mount -fv -t proc proc /proc || ECHO_RETURN=$rc_failed_up
+mount -v /S.u.S.E. || ECHO_RETURN=$rc_failed_up
 mount -av -t nonfs,noproc || ECHO_RETURN=$rc_failed_up
 echo -e "$ECHO_RETURN"
 
@@ -373,11 +373,13 @@
 	esac
 done < /proc/cpuinfo
 
-test "$GMT" != "YAST_ASK" && $CLOCKCMD --hctosys $GMT || ECHO_RETURN=$rc_failed
-test -f /etc/adjtime || echo "0.0 0 0.0" > /etc/adjtime
-test "$GMT" != "YAST_ASK"  -a "$START_XNTPD" != "yes" && {
-    $CLOCKCMD --adjust $GMT || ECHO_RETURN=$rc_failed
-}
+if [ "$GMT" != YAST_ASK ] ; then
+  test "$GMT" != "YAST_ASK" && $CLOCKCMD --hctosys $GMT || ECHO_RETURN=$rc_failed
+  test -f /etc/adjtime || echo "0.0 0 0.0" > /etc/adjtime
+  test "$GMT" != "YAST_ASK"  -a "$START_XNTPD" != "yes" && {
+      $CLOCKCMD --adjust $GMT || ECHO_RETURN=$rc_failed
+  }
+fi
 echo -e "$ECHO_RETURN"
 
 
